sig-observability/concepts-and-designs/notification-manager-v3.1.0.md

```diff
@@ -61,7 +61,7 @@
 
 ## Demand ##
 
-- User can define the channels for receiving notifications, including email¡¢wechat¡¢slack¡¢webhook¡¢dingtalk,  by creating the corresponding receiver and config.
+- User can define the channels for receiving notifications, including email,wechat,slack,webhook,dingtalk, by creating the corresponding receiver and config.
```

Comment:
including email,wechat,slack,webhook,dingtalk, by creating the =>
including email,wechat,slack,webhook and dingtalk, by creating the